### PR TITLE
Improvements of consistency for post-hoc hypothesis test tidiers

### PR DESCRIPTION
**Pull request by crsh**
## Description

This PR implements the changes discussed in #692, i.e.

### `emmeans`

- [x] add the arguments `conf.int` and `conf.level`
- [x] if applicable, change `p.value` to `adj.p.value`
- [x] if applicable, add `term` column
- [x] use `contrast` (with `A - B`) and `null.value` columns

### `multcomp`

- [x] include `summary()`-call
- [x] if applicable, change `p.value` to `adj.p.value`
- [x] if applicable, add `term` column
- [x] use `contrast` (with `A - B`) and `null.value` columns

### `TukeyHSD`

- [x] use `contrast` (with `A - B`) and `null.value` columns

Some of these are breaking changes, as I've tried to summarize in `NEWS`

> Tidiers for `emmeans`-objects use the arguments `conf.int` and `conf.level`
    instead of relying on the argument names native to the `emmeans::summary()`-
    methods (i.e., `infer` and `level`). Similarly, `multcomp`-tidiers now
    include a call to `summary()` as previous behavior was akin to passing
    `quick = TRUE`. Both families of tidiers now use the `adj.p.value` column
    name when appropriate. Finally, `emmeans`-, `multcomp`-, and `TukeyHSD`-
    tidiers now consistently use the column names `contrast` and `null.value`
    instead of `comparison`, `level1` and `level2`, or `lhs` and `rhs` (see #692).

## Note

Currently `strict = FALSE` is needed for `check_tidy_output()` in all tests because `null.value` is not in the column glossary.

### CODE REVIEW

File: R/multcomp-tidiers.R [hunk 1]
```diff
@@ -36,12 +36,22 @@
...
-    lhs = rownames(x$linfct),
-    rhs = x$rhs,
-    estimate = stats::coef(x)
-  )
+tidy.glht <- function(x, conf.int = FALSE, conf.level = .95, ...) {
+  glht_summary <- summary(x, ...)
+  
+  tidy_glht_summary <- tidy.summary.glht(glht_summary, ...)
+  
+  if(isTRUE(conf.int)) {
```
Comment (alexpghayes) [inline R/multcomp-tidiers.R]:
```suggestion
  if (conf.int) {
```

**Comment by crsh** [comment]
Thanks, I committed your suggestion and resolved recently introduced merge conflicts.

**Comment by crsh** [comment]
Hi @alexpghayes, sorry to bother you again with this. Is there anything you need from me to decide on whether this PR can be merged? The development of some portion of a package of mine is depends on these changes.

**Comment by alexpghayes** [comment]
Apologies for the long delay on this! Thank you for all your work and patience!

**Pull request merged**